.coveragerc

```diff
@@ -1,8 +1,17 @@
 # .coveragerc to control coverage.py
 
+[run]
+omit =
+    config/*
+    flask/*
+    instance/*
+    migrations/*
+    sourcemash/frontend/*
+    tests/*
```

Comment:
Yeah. If you omit in coverage, it still runs the tests; it just doesn't return the % covered of the files in those folders.

We don't want to track 100% coverage on our test code...or on entire Python libraries for that matter...